diff --git a/routers/api/v1/repo/repo.go b/routers/api/v1/repo/repo.go
index 62153893a..81ab68e29 100644
--- a/routers/api/v1/repo/repo.go
+++ b/routers/api/v1/repo/repo.go
@@ -295,6 +295,25 @@ func CreateOrgRepo(ctx *context.APIContext, opt api.CreateRepoOption) {
 			ctx.ServerError("IsOwnedBy", err)
 			return
 		} else if !isOwner {
+			isMember, err := org.IsOrgMember(ctx.User.ID)
+			if err != nil {
+				ctx.ServerError("IsOrgMember", err)
+				return
+			}
+			if isMember {
+				teams, err := models.GetUserOrgTeams(org.ID, ctx.User.ID)
+				if err != nil {
+					ctx.Error(500, "GetUserOrgTeams", err)
+					return
+				}
+
+				for _, team := range teams {
+					if team.HasWriteAccess() {
+						CreateUserRepo(ctx, org, opt)
+						return
+					}
+				}
+			}
 			ctx.Error(403, "", "Given user is not owner of organization.")
 			return
 		}
